bootstrap/map.jinja

```diff
@@ -5,7 +5,7 @@
        'grub_install_device': '/dev/' + (salt['grains.get']('SSDs:0') or 'sda'),
        'kernel': {
          'meta_package': 'linux-image-generic-lts-trusty',
-         'version': '3.13.0.57.49'
+         'version': '3.13.0.58.65'
```

Comment:
The version for 12.04/precise is different: http://packages.ubuntu.com/precise/linux-image-generic-lts-trusty -- 3.13.0.58.50